src/model.py

```diff
@@ -41,6 +41,49 @@
 
 
 class TritonPythonModel:
+    @staticmethod
+    def auto_complete_config(auto_complete_model_config):
+        inputs = [
+            {"name": "text_input", "data_type": "TYPE_STRING", "dims": [1]},
+            {"name": "stream", "data_type": "TYPE_BOOL", "dims": [1]},
```

Comment:
stream should be optional too IMO. I think it was in the tutorial.

Reply:
I'll double check and update

Reply:
Nice catch, for some reason during the migration from the tutorial to here, `optional` in "stream" got lost, I'll update accordingly.

Reply:
added optional strem